docs/monorepo.md

```diff
@@ -86,6 +91,45 @@ To run one package's tests:
 
 `npm run test-packages [ test file pattern ]`
 
+## Building packages & apps
+
+Packages will have their `prepare` scripts run automatically on `npm install`.
+
+You can build packages also by running:
+
+```bash
+npm run build-packages
+```
+
+Or even specific packages:
+
+```bash
+npx lerna run prepare --scope="@automattic/calypso-build"
+```
+
+Or specific apps:
+
+```bash
+npx lerna run build --scope="@automattic/calypso-build"
+```
+
+All `prepare` scripts found in all `package.json`s of apps and packages are always run on Calypso's `npm install`. Therefore independent apps in `/apps` directory can use `build` instead of `prepare` so avoid unnecessary builds.
+
+You can also run other custom `package.json` scripts only for your app or package:
+```bash
+npx lerna run your-script --scope="@automattic/your-package"
+```
+
+## Developing packages
+
+When developing packages in Calypso repository that external consumers (like Jetpack repository) depend on, you might want to test them without going through the publishing flow first.
+
+1. Enter the package you're testing
+1. Run [`npm link`](https://docs.npmjs.com/cli/link) — the package will be installed on global scope and symlinked to the folder in Calypso
+1. Enter the consumer's folder (such as Jetpack)
+1. Type `npm link @automattic/package-name` — the package will be symlinked between Calypso and Jetpack and any modifications you make in Calypso, will show up in Jetpack.
```

Comment:
Maybe add a note that if you're building with webpack, you may need to turn off [resolve.symlinks](https://webpack.js.org/configuration/resolve/#resolvesymlinks) for it to work as expected.

Reply:
Good point. Added in https://github.com/Automattic/wp-calypso/pull/32137/commits/b76f2641cd9f3eb9b2458cc60ea7e4c46691d594